py/kubeflow/testing/util.py

```diff
@@ -831,6 +843,16 @@ def maybe_activate_service_account():
     logging.info("GOOGLE_APPLICATION_CREDENTIALS is not set.")
 
 
+def aws_configure_credential():
+  if "Unable to locate credentials" not in run(["aws", "eks", "list-clusters"]):
```

Comment:
Why not use boto3 SDK? using commands in python is not best practice

Reply:
The reason I don't use boto3 SDK is: I don't want to mix commands and sdk up. I want to either use SDK for all processes or commands.

But there are some commands that not support yet in boto3, such as there's no SDK way to `aws eks update-kubeconfig --name=XXXX`.